### PR TITLE
21x updates sponsors

### PR DESCRIPTION
**Pull request by carlynlee**
## Description of PR
relates to: #63 

updated sponsors for 21x

### CODE REVIEW

**Comment by irabinovitch** [comment]
Thanks @carlynlee. How should we let you know about additional sponsors as they sign up?

**Comment by sarcasticadmin** [comment]
@irabinovitch how about an github issue with a checklist? Then just append to the list as new sponsors get added and its easy to see which ones are new

File: server/sponsors.php [hunk 1]
```diff
@@ -3,134 +3,67 @@
...
-          "kubeevents.png",
-          "kloudfuse.png",
-          "linode.png",
-          "linbit.png",
-          "loft.png",
-          #"lpi.png",
-          "mattermost.png",
-          "meta.png", #facebook
-          "maira.png",
+             "arm.png",
```
Comment (sarcasticadmin) [inline server/sponsors.php:6]: Thanks for balancing this out!

File: server/room.php [hunk 2]
```diff
@@ -69,87 +69,64 @@
 );
 
 $sponsors = array(
```
Comment (sarcasticadmin) [inline server/room.php:71]: this array is looking good!

**Review by sarcasticadmin — APPROVED** [review]
@carlynlee great job getting these changes together! :shipit:

**Pull request merged**